chain/beacon/cache.go

```diff
@@ -110,18 +110,20 @@ func (c *partialCache) getCache(id string, p *drand.PartialBeaconPacket) *roundC
 }
 
 type roundCache struct {
-	round uint64
-	prev  []byte
-	id    string
-	sigs  map[int][]byte
+	round  uint64
+	prev   []byte
+	id     string
+	sigs   map[int][]byte
```

Comment:
is this preferable to
```
type v2Sig struct {
  sig []byte
  sigv2 []byte
}

sigs map[int]v2Sig
```

Reply:
Given that the struct is not necessarilly filled with both sigs I wanted to keep them separate. Matter of taste I suppose or is there anything more to it ?